kind-local/dockerhost.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: dockerhost
+subsets:
+- addresses:
+  - ip: @GATEWAY_IP@ # this is the gateway IP in the "kind" docker network
```

Comment:
This is the trick let agents running in Kind to connect to the controller running in the host.